src/Garcia/Router.php

```diff
@@ -43,18 +56,20 @@ public static function resource(string $path, string $className)
         self::addRoute('PATCH', "$path/:id", fn ($params) => (new $className)->update($params));
         self::addRoute('PUT', "$path/:id", fn ($params) => (new $className)->update($params));
         self::addRoute('DELETE', "$path/:id", fn ($params) => (new $className)->destroy($params));
+        return new static;
     }
      
     /**
      * Sets a new route for GET requests.
      *
      * @param string $path - URL path
      * @param callable $handler - Route handler
-     * @return void
+     * @return object
      */
     public static function get(string $path, callable $handler)
     {
         self::addRoute('GET', $path, $handler);
+         return new static;
```

Comment:
This is good, please at the `return new static` statement to all the other request: `post`, `put`, etc. and update the return comments accordingly. Thank you!

Reply:
fixed it